core/frontend/src/tile/map/ArcGisUtilities.ts

```diff
@@ -140,6 +140,19 @@ export class ArcGisUtilities {
         return { status: MapLayerSourceStatus.InvalidCredentials, authInfo: { authMethod: MapLayerAuthType.EsriToken } };
     }
 
+    // Check this service support map queries
+    let hasMapCapability = false;
+    try {
+      if (json.capabilities
+        && typeof json.capabilities === "string"
+        && json.capabilities.toLowerCase().includes("map")) {
```

Comment:
What REST API you are invoking and is there a JSON schema for the response?

Reply:
When tiles are fetched, two API can be used depending on the service capability response:
1. [Export Map](https://developers.arcgis.com/rest/services-reference/enterprise/export-map.htm)
2. [Map Tile](https://developers.arcgis.com/rest/services-reference/enterprise/map-tile.htm)

The response format is an image.

Reply:
I'm interested specifically in the format of the string provided by `json.capabilities`. e.g., is it a delimited list of capability names? Is checking for the substring "map" anywhere in that string sufficient?

Reply:
On the [Map Service](https://developers.arcgis.com/rest/services-reference/enterprise/map-service.htm) page, there is a _JSON response syntax_ section, that explicitly mentions:

` "capabilities": "<capabilities>",  //comma separated list of supported capabilities - e.g. "Map,Query,Data"`

I'm doing case-insensitive comparison just to be on the safe side, I think we could simply check for 'Map'.  Btw, ArcGISMapLayerImageryProvider we already check for capabilities, such as `Query` or `Tilemap`